editor/code_editor.cpp

```diff
@@ -481,6 +481,9 @@ void FindReplaceBar::set_text_edit(TextEdit *p_text_edit) {
 void FindReplaceBar::_update_size() {
 
 	container->set_size(Size2(hbc->get_size().width, 1));
+
+	// update variable tab_container in script_editor_plugin
+	get_parent_control()->get_parent_control()->get_parent_control()->minimum_size_changed();
```

Comment:
Such an hardcoded call to the grand-grandparent of the node doesn't seem like the right fix. Usually such changes are propagated with signals.

Reply:
Yeah, I agree that this is weird but to emit a signal, if I remember well, I'll have to create an signal on FindReplaceBar, CodeTextEditor, ScriptTextEditor to finally connect it on ScriptEditorPlugin to update tab_container. Unless there is a better way of propagating that I haven't realized.

I'll give a check on it.